.circleci/config.yml

```diff
@@ -16,14 +16,14 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}
+            - v2-dependencies-{{ checksum "package.json" }}
```

Comment:
This is how we get CircleCI to stop using old versions of cache, according to the docs